modules/age.nix

```diff
@@ -33,8 +35,18 @@ let
     )
     chmod ${secretType.mode} "$TMP_FILE"
     chown ${secretType.owner}:${secretType.group} "$TMP_FILE"
+
+    # see if there's been any change from the last generation
+    changes=$(${pkgs.rsync}/bin/rsync --dry-run -aHAX -i "$TMP_FILE" "$_oldPath")
```

Comment:
I'm also a little worried about the dependency on rsync. Is that already included in the default system closure?